contribute/index.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: std
+title: Enter the void - Contribute
+---
+
+<div>
+<h2>Contribute</h2>
+ <p>To help with adding or updating Void packages, start by visiting <a href= "https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md">CONTRIBUTING</a>.
+ <p>To help with adding or updating <a href="https://docs.voidlinux.org/">the documentation</a>, start by visiting <a href="https://docs.voidlinux.org/contributing/void-docs/index.html">this section of the Void Handbook</a>.
+ <p>If you have any questions, ask on <a href="irc://irc.freenode.net/#voidlinux">IRC</a> or <a href="https://www.reddit.com/r/voidlinux">the forum</a>.
```

Comment:
IRC link handlers aren't as widespread as they probably should be, consider stating the channel and network here rather than a blind link so people don't have to get this information from a hove over.

Reply:
Sure, will do.

Reply:
Also, the IRC link on the home page follows the pattern i'd used here - is that something that should be addressed (in a separate PR, obviously)?

Reply:
It would definitely make the front page more accessible, and it would be easier for everyone to find.

Reply:
Go ahead and open a PR for the home page so it can be discussed, other maintainers may have stronger opinions about the home page itself.

Reply:
Thanks, [done](https://github.com/void-linux/void-linux.github.io/pull/87).